README.md

```diff
@@ -97,6 +97,9 @@ By default, `ENABLE_AUTHENTICATION` is set to off/`false`. To enable external Ve
     2. `LOGTO_WEBHOOK_SECRET`: Webhook secret to authenticate incoming webhook requests from LogTo.
 5. **Miscellaneous**
     1. `COOKIE_SECRET`: Secret for cookie encryption.
+    2. `API_KEY_PREFIX` (optional): Prefix for API keys. (Default "caas")
+    3. `API_KEY_LENGTH` (optional): Length of API keys. (Default 32)
```

Comment:
Just use a default. No need to make this configurable

Reply:
What key length does Cloudflare use? I'd use similar (I believe they are either 40 or 64)